mobile/src/main/java/org/openhab/habdroid/core/OpenHABTracker.java

```diff
@@ -95,7 +95,7 @@ public void start() {
                     // If remote URL is configured
                     if (mOpenHABUrl.length() > 0) {
                         Log.d(TAG, "Connecting to remote URL " + mOpenHABUrl);
-                        openHABTracked(mOpenHABUrl, mCtx.getString(R.string.info_conn_rem_url));
```

Comment:
Can we modify the PR to not complain about the voice recognizer but leave the other messages?